src/mono/wasm/runtime/loader/blazor/_Integration.ts

```diff
@@ -22,7 +22,10 @@ export async function loadBootConfig(config: MonoConfigInternal, module: DotnetM
 export async function initializeBootConfig(bootConfigResult: BootConfigResult, module: DotnetModuleInternal, startupOptions?: Partial<WebAssemblyStartOptions>) {
     INTERNAL.resourceLoader = resourceLoader = await WebAssemblyResourceLoader.initAsync(bootConfigResult.bootConfig, startupOptions ?? {});
     mapBootConfigToMonoConfig(loaderHelpers.config, bootConfigResult.applicationEnvironment);
-    setupModuleForBlazor(module);
+
+    if (ENVIRONMENT_IS_WEB) {
```

Comment:
Because the ResourceLoader that came from blazor uses directly fetch https://github.com/dotnet/runtime/blob/main/src/mono/wasm/runtime/loader/blazor/WebAssemblyResourceLoader.ts#L152

In the follow up PRs I'm going to unify typescript code and dissolve this blazor integration layer.